capi/src/visitors/fragment-tags.ts

```diff
@@ -1,13 +1,25 @@
 import * as t from "../types";
 import clone from "clone-deep";
 
+const isPContainingFragments = function(node: t.HyperscriptNode): boolean {
+  if (!Array.isArray(node)) return false;
+  if (node[0] !== "p") return false;
+  if (node.length < 3) return false;
+  const firstChild = node[2];
```

Comment:
Is this the first child?

Reply:
Yes, the structure goes like this: `[<node name>, <map of node properties, like class, id, onhover, or whatever>, <child 1>, <child 2>, ...]`.  Maybe there's some TS magic you can do to make this more clear but I couldn't think of anything.